app/crud/base.py

```diff
@@ -34,7 +34,7 @@ def create(self, obj: CreateSchemaType) -> ModelType:
         except sqlalchemy.exc.IntegrityError as e:
             self.db_session.rollback()
             if "duplicate key" in str(e):
-                raise ExceptionCustom(status_code=409, detail="Conflict Error")
+                raise ExceptionCustom(status_code=409, detail="Duplicated Key, Name field is Duplicated")
```

Comment:
praise: thank you for paying attention to this Exception!
suggestion: extract the information of the field from the error object itself, don't hardcode it